dist/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Todo list App</title>
+<script defer src="/todo-list-app/main.js"></script></head>
+
+<body>
+
+    <div class="wrapper-container">
+        <div class="wrapper">
+            <div class="top">
+                <h2>Today's To Do</h2>
+                <button type="submit"><i class="fas fa-sync fa-xs"></i></button>
+            </div>
+
+
+            <form id="form">
+                <input type="text" placeholder="Add to your list..." class="text">
+                <button type="submit"><i class="fa fa-reply"></i></button>
+            </form>
+
+            <ul id="todoList">
+
+            </ul>
+
+            <footer>
+                <span class="clear-completed">Clear all completed</span>
+            </footer>
+
+        </div>
+
+    </div>
+
+</body>
+
+</html>
```

Comment:
Please do not add blank lines or spaces without a reason. Add blank lines to separate large or logical code blocks. 💪